platform/ui/src/views/StudyList/components/StudyListPagination.js

```diff
@@ -39,21 +50,24 @@ const StudyListPagination = props => {
               <ButtonGroup color="primary">
                 <IconButton
                   size="small"
-                  className="border-custom-darkSlateBlue px-4"
+                  className="border-custom-darkSlateBlue px-4 py-2 text-base"
+                  style={{ padding: '3px 12px', minWidth: 0 }}
```

Comment:
I'm not 100% comfortable to have inline styles. This is a specific/temporary case. Just to keep in mind that ideally, it shouldn't be done. 🙄

Reply:
No worries. If we're forced to do this, worth mentioning to Dan. Especially if this doesn't fit the same padding increments as the rest of the design. Also, select's are hard to style